app/forms/idv/doc_pii_form.rb

```diff
@@ -18,9 +18,10 @@ class DocPiiForm
     validates_presence_of :state_id_number, { message: proc {
       I18n.t('doc_auth.errors.general.no_liveness')
     } }
+    validate :expired?
```

Comment:
I might rename this to make it a little clearer what's being checked?
```suggestion
    validate :state_id_expiration?
```

Reply:
Adding on to this comment:

- The method doesn't return a boolean, so I wouldn't expect a `?` suffix
- Most of our "validate" methods are prefixed with `validate_`

```suggestion
    validate :validate_state_id_expiration
```